smart-contracts/test/test_txSignatureAggregation.js

```diff
@@ -60,9 +63,15 @@ describe("Gas Cost Tests", function () {
       owner,
       userOne,
       userThree,
-      pauser
+      pauser,
+      networkDescriptor
     );
 
+    // Add the token into white list
+    await state.bridgeBank.connect(operator)
+    .updateEthWhiteList(state.token1.address, true)
```

Comment:
Please change this whole file to indent statements that extend from previous lines.